qemu/tests/block_with_iommu.py

```diff
@@ -60,6 +61,10 @@ def verify_iommu_enabled():
     session = vm.wait_for_login(timeout=360)
     verify_iommu_enabled()
 
+    session.cmd(params.get("dd_cmd"))
+
     if params.get("reload_kernel_cmd"):
         reload_kernel(session)
-        vm.reboot(session)
+
+    vm.reboot(session)
+    vm.verify_alive()
```

Comment:
suggest change to session = vm.wait_for_login(timeout=360). verify_alive just indicates the pid is exist,but does not prove the guest is workable, and subsequent action needs the session

Reply:
Updated !